CONTRIBUTING.rst

```diff
@@ -22,7 +22,7 @@ In order to add a feature:
   documentation.
 
 - The feature must work fully on the following CPython versions:
-  3.7, 3.8, 3.9, 3.10, 3.11 and 3.12 on both UNIX and Windows.
+  3.7, 3.8, 3.11 and 3.12 on both UNIX and Windows.
```

Comment:
Curious: why the de-emphasis on versions 3.9 and 3.10?

Reply:
I think this is taking the unit test versions from owlbot. We do want it to work on these, but I wonder if it's worth testing against so many versions?